benchmark/main.cpp

```diff
@@ -560,7 +570,11 @@ MBED_NOINLINE static int benchmark_aes_ccm()
         memset(tmp, 0, sizeof(tmp));
 
         ret = mbedtls_ccm_setkey(&ccm, MBEDTLS_CIPHER_ID_AES, tmp, keysize);
-        if (ret != 0) {
+        if (ret == MBEDTLS_ERR_AES_FEATURE_UNAVAILABLE) {
```

Comment:
Note: Obviously this will work only for AES feature unavailable, and not in case an alternative ccm implementation returns the uncreated yet `MBEDTLS_CCM_FEATURE_UNAVAILABLE`

This is not a review comment, but an notice for future implementation

Reply:
I have expanded the documentation in the code to explain that the checks need to be extended as more FEATURE_UNAVAILABLE errors are added to the library.